### PR TITLE
add module deployment doc in English

### PR DESCRIPTION
**Pull request by guoquanwu**
**What does this PR do?**

add module deployment doc in English

### CODE REVIEW

**Comment by guoquanwu** [comment]
no code changed, maybe rerun will be ok, how to rerun?

File: docs/modular-deployment-en.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# How to deploy java-tron after modularization
+
+After modularization, java-tron is launched via shell script instead of typing command: `java -jar FullNode.jar`.
+
+*`java -jar FullNode.jar` still works, but will be deprecated in future*
```
Comment (andelf) [inline docs/modular-deployment-en.md]: missing period

File: docs/modular-deployment-en.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Download
+
+```
+git clone git@github.com:tronprotocol/java-tron.git
+```
+
+## Compile
+
+cd project dictionary:
```
Comment (andelf) [inline docs/modular-deployment-en.md]: `s/cd/change directory to/`

File: docs/modular-deployment-en.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+java-tron-1.0.0.zip will be generated in java-tron/build/distributions after compilation
+
+## Unzip
+
+Unzip java-tron-1.0.0.zip
+```
+cd java-tron/build/distributions
+unzip -o java-tron-1.0.0.zip
+```
+After unzip, two dictionaries will be generated in java-tron: `bin` and `lib`, shell scripts are located in `bin`, jars are located in `lib`
```
Comment (andelf) [inline docs/modular-deployment-en.md]: `s/dictionaries/directories/`

File: docs/modular-deployment-en.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+java-tron-1.0.0/bin/FullNode
+
+# using config file, there are some demo configs in java-tron/framework/build/resources
+java-tron-1.0.0/bin/FullNode -c config.conf
+
+# when startup with SR mode，add parameter: -w
+java-tron-1.0.0/bin/FullNode -c config.conf -w
+```
+
+## Jvm config
```
Comment (andelf) [inline docs/modular-deployment-en.md]: `s/Jvm/JVM/`

File: docs/modular-deployment-en.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+Jvm options can also be customized in java-tron, located in bin/java-tron.vmoptions:
```
Comment (andelf) [inline docs/modular-deployment-en.md]: wrap filename into a

    `path/to/your_file`

Comment (andelf) [reply docs/modular-deployment-en.md]: missing subject part of the second half, which causes ambiguity

File: docs/modular-deployment-en.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+Unzip java-tron-1.0.0.zip
+```
+cd java-tron/build/distributions
+unzip -o java-tron-1.0.0.zip
+```
+After unzip, two dictionaries will be generated in java-tron: `bin` and `lib`, shell scripts are located in `bin`, jars are located in `lib`
+
+## Startup
+
+Using the corresponding script to start java-tron according to the OS type, using `*.bat` on Windows, Linux demo is as below:
```
Comment (andelf) [inline docs/modular-deployment-en.md]: `s/using/use/g`, they are imperative sentences

File: docs/modular-deployment-en.md (hunk 3 above)
Comment (andelf) [reply docs/modular-deployment-en.md]: trailing period

File: docs/modular-deployment-en.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+git clone git@github.com:tronprotocol/java-tron.git
+```
+
+## Compile
+
+cd project dictionary:
+```
+./gradlew build
+```
+java-tron-1.0.0.zip will be generated in java-tron/build/distributions after compilation
```
Comment (andelf) [inline docs/modular-deployment-en.md]: trailing period

**Comment by andelf** [comment]
FYI:

How to re-run CI
> log in https://travis-ci.org/ with your GitHub account. the re-run button will be available.

**Comment by andelf** [comment]
Since your pr contains only plain markdown file,
you can use the `[skip ci]` trick.

ref: <https://docs.travis-ci.com/user/customizing-the-build#skipping-a-build>

File: docs/modular-deployment-en.md (hunk 1 above)
Comment (guoquanwu) [reply docs/modular-deployment-en.md]: solved

File: docs/modular-deployment-en.md (hunk 2 above)
Comment (guoquanwu) [reply docs/modular-deployment-en.md]: solved

File: docs/modular-deployment-en.md (hunk 3 above)
Comment (guoquanwu) [reply docs/modular-deployment-en.md]: solved

File: docs/modular-deployment-en.md (hunk 4 above)
Comment (guoquanwu) [reply docs/modular-deployment-en.md]: solved

File: docs/modular-deployment-en.md (hunk 5 above)
Comment (guoquanwu) [reply docs/modular-deployment-en.md]: solved

File: docs/modular-deployment-en.md (hunk 6 above)
Comment (guoquanwu) [reply docs/modular-deployment-en.md]: solved

File: docs/modular-deployment-en.md (hunk 7 above)
Comment (guoquanwu) [reply docs/modular-deployment-en.md]: solved

**Pull request merged**